SCons/Script/__init__.py

```diff
@@ -390,14 +390,13 @@ def Variables(files=None, args=ARGUMENTS):
 SConscript = _SConscript.DefaultEnvironmentCall('SConscript')
 
 # Make SConscript look enough like the module it used to be so
-# that pychecker doesn't barf.
-SConscript.__name__ = 'SConscript'
-
-SConscript.Arguments = ARGUMENTS
-SConscript.ArgList = ARGLIST
-SConscript.BuildTargets = BUILD_TARGETS
-SConscript.CommandLineTargets = COMMAND_LINE_TARGETS
-SConscript.DefaultTargets = DEFAULT_TARGETS
+# that pychecker and mypy don't barf.
```

Comment:
the checkers don't like the previous syntax?

Reply:
Correct. They gripe that there's no such attribute. Obviously Python allows you to add them this way, but it seemed a cheap change to shut the tools up.